### DISCUSSION TITLE
What is the TX and RX?

### DISCUSSION THREAD

**Discussion by designe** [euphony-io/euphony#48] [Knowledge Share]
![Screen Shot 2021-09-16 at 12 30 49 AM](https://user-images.githubusercontent.com/1748714/133463545-fc626ce8-dcea-47af-a8ae-814dfb0bb51a.png)

There are terms such as TX and RX that you often hear 
when studying subjects like digital communication or embedded development. 

![image](https://user-images.githubusercontent.com/1748714/133462709-59f78bb3-cc0b-4e37-8c33-39a5a6b2b658.png)

In the field of communication, 
**TX means Transmit and RX means Receive**

Well. I understand T and R, and what is X?

I was curious too, so I asked a lot of my friends in the electronics department, 
but most of them didn't seem to know. 

So I just googled and found that there is really no meaning in X. 
As you know that, Airports are usually expressed in three letters. 
Incheon Airport is ICN, 
Gimpo Airport is GMP, etc.

LA Airport is called LAX, and an X is added to match it with three letters, 
so it can be considered similar to TX and RX.

**Comment by kmsaidev** [comment]
I was also curious about that. Thank you for explaining it well.

**Comment by ycs1m1yk** [comment]
Yes it is,  -x is used as abbreviation maker.

https://en.wiktionary.org/wiki/-x#See_also

<img width="500" alt="abbreviation-maker" src="https://user-images.githubusercontent.com/27720475/135805464-677395cc-e71f-4e9e-97db-346ff6f55c96.png">

**Reply by aiclaudev** [reply]
So interesting!!

**Comment by kuro11pow2** [comment]
That was the question I was curious about! Thank you for the explanation.

**Comment by psh2849** [comment]
Interesting! Thank you !! 😄
